nixos/modules/system/boot/unl0kr.nix

```diff
@@ -0,0 +1,91 @@
+{ config, lib, pkgs, utils, ... }:
+
+with lib;
+
```

Comment:
```suggestion
```

Minor nit, but having a `with` with such a wide scope should be avoided... It makes it harder down the line to reason where things come from, especially for newcomers. (Though less so when only a single `with` is at play...)

Whether you always use `lib.___`  or use `inherit (lib) ___` in the following `let` block is up to your preferences I guess.